app/assets/src/components/ResultsFolder.jsx

```diff
@@ -1,12 +1,123 @@
 import React from "react";
 import Divider from "./layout/Divider";
 
+class OutputFile extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this.file = props.file;
+  }
+
+  download(url) {
+    if (url) {
+      location.href = `${url}`;
+    }
+  }
+
+  render() {
+    return (
+      <tr
+        className={`${this.file.url ? "" : "disabled-"}file-link`}
+        onClick={this.download.bind(this, this.file.url)}
```

Comment:
For standardization this should use utils/links.js

Reply:
ok moved